Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -84,6 +84,8 @@ def bottle_args
   end
 
   def bottle
+    ENV["HOMEBREW_NO_PATCHELF_RB_WRITE"] = "1" unless ENV["HOMEBREW_PATCHELF_RB_WRITE"].present?
```

Comment:
Was thinking this could go into `brew test-bot`?

Reply:
Done!